ocp_resources/security_context_constraints.py

```diff
@@ -3,7 +3,196 @@
 
 class SecurityContextConstraints(Resource):
     """
-    Security Context Constraints object.
+    Security Context Constraints object. API reference:
+    https://docs.openshift.com/container-platform/4.13/rest_api/security_apis/securitycontextconstraints-security-openshift-io-v1.html
+
+    Args:
```

Comment:
1. should be under __init__
2. do not repeat the api docs
3. defaults by the api should not appear here

Reply:
Done